core/CMake_globalize_pic_thunks.cmake

```diff
@@ -30,16 +30,19 @@
 # DAMAGE.
 
 # Caller must set:
+# + "lib_fileloc" to <name> that represents a file <name>.cmake the contains a
+# set(<name> <path>) where path is a library location. The .cmake file must reside
+# in the root build directory.
 # + CMAKE_C_COMPILER to point to CC compiler
 # + partial_link_flags to linker flags incl. -m32/-m64
 # + disable_pie_flag to -no-pie or ""
 # + CMAKE_OBJCOPY to point to objcopy
 # + CMAKE_AR to point to ar
 # + CMAKE_RANLIB to point to ranlib
 
-include(../../dynamorio_static_loc.cmake)
+include(../../${lib_fileloc}.cmake)
```

Comment:
But this is still making an assumption on its working directory: how do you know it's always going to be run 2 levels deep?  The point of the absolute path is to avoid these kinds of assumptions.  Just pass in the absolute path and include it directly here.  If you want to keep the filename==varname and not have a separate var for the varname, get the basename of the abs path here as the varname.

Ditto below.

Reply:
Done